mobile/lib/blocs/nearby_location/nearby_location_bloc.dart

```diff
@@ -109,7 +107,7 @@ class NearbyLocationBloc
       position: event.position,
     );
 
-    airQualityReadings = airQualityReadings.sortByDistanceToReferenceSite();
+    airQualityReadings = airQualityReadings..sortByDistanceToReferenceSite();
 
```

Comment:
Is this supposed to single calling "dot" or a double "dot"? @NoahNsimbe

Reply:
this should be a single dot since we are not perform multiple operations. Thanks @Baalmart